syntax = "proto3";
package GameEngine;

import "InitBoardReq.proto";
import "CloseEngineCommand.proto";
import "StartEngine.proto";

message ReqWrapper
{
	oneof msg
	{
		InitBoardReq initBoardReq = 1;
		CloseEngineCommand closeEngineCommand = 2;
		StartEngine startEngine = 3;
	}
}